File: sudosumo_backend/src/main/java/io/github/louisp78/sudosumo_backend/domain/UserDomain.java
package io.github.louisp78.sudosumo_backend.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDomain {
    private Long id;
    private String token;
    private Integer score = 0;
    private String email;
    private String username;
    private String firstName;
    private String lastName;
}

File: sudosumo_blabla/src/domain/UserDomain.ts
class UserDomain {
    id: number;
    score: number;
    firstName: string;
    lastName: string;
    email: string;
    username: string;

    constructor(id: number, score: number, firstName: string, lastName: string, email: string, username: string) {
        this.id = id;
        this.score = score;
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.username = username;
    }
}